src/assets/RustAsset.js

```diff
@@ -153,7 +153,8 @@ class RustAsset extends Asset {
 
     // Get output file paths
     let outDir = path.join(cargoDir, 'target', RUST_TARGET, 'release');
-    let rustName = cargoConfig.package.name;
+    // Rust converts '-' to '_' when outputting files.
+    let rustName = cargoConfig.package.name.replace('-', '_');
```

Comment:
This will only replace the first dash. We should make sure to replace every instance.